morerss/zhihu_stream.py

```diff
@@ -122,14 +182,21 @@ def post2rss(post, digest=False, pic=None):
     logger.warn('unknown type: %s', post['type'])
     return
 
-  if digest:
+  # Posts in Zhihu topics API response don't have the 'content' key by default
+  # Although they can include it by carrying verbose query params
+  # which are hard to maintain because Zhihu doesn't have public API documentation :(
+  if 'content' not in post or digest:
     content = post['excerpt']
   else:
     content = post['content']
 
   content = content.replace('<code ', '<pre><code ')
   content = content.replace('</code>', '</code></pre>')
```

Comment:
6d01a6e also addresses some posts that only contain images but no text in the Zhihu topics API.